### PR TITLE
Vendor ie11

### LINKED ISSUE #271: Broken in IE11 as of version 4.1.0
The addition of https://github.com/kellyselden/ember-macro-helpers/blob/v4.1.0/vendor/monkey-patch-ember-computed.js in version `4.1.0` seems to be breaking this addon in IE11 due to that file containing ES6 code that isn't being compiled by `ember-cli-babel`.

If required I can setup a reproduction repo to demonstrate the issue, but here are the basic steps to verify the problem:
- Create a new ember app with `ember-cli` (I used `ember-cli@3.11.0`): `ember new my-app`
- Update the `config/targets.js` file to include IE11 by default:
```
diff --git a/config/targets.js b/config/targets.js
index 8ffae36..03c1395 100644
--- a/config/targets.js
+++ b/config/targets.js
@@ -3,16 +3,10 @@
 const browsers = [
   'last 1 Chrome versions',
   'last 1 Firefox versions',
-  'last 1 Safari versions'
+  'last 1 Safari versions',
+  'ie 11'
 ];

-const isCI = !!process.env.CI;
-const isProduction = process.env.EMBER_ENV === 'production';
-
-if (isCI || isProduction) {
-  browsers.push('ie 11');
-}
-
```
- Add `ember-macro-helpers`: `ember install ember-macro-helpers`
- Serve the app and try viewing in IE11: `ember serve`
- A syntax error is shown in console for the monkey patch file

It'd be great if this addon could continue to support IE11, but if not then the incompatibility should probably be added to the README to let people know that the latest version doesn't support IE.

### PR DESCRIPTION
**Pull request by kellyselden**
Closes  #271.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "renovate-config-standard": "^2.0.0"
   },
   "engines": {
-    "node": "8.* || >= 10.*"
```
Comment (gorner) [inline package.json]: Why was this needed? (This broke a CI build for us running Node v8.16.)

File: package.json [hunk 2]
```diff
@@ -57,7 +57,7 @@
...
+    "node": "8.3 || >= 10.*"
```
Comment (gorner) [inline package.json]: Why was locking to 8.3 needed? (This broke a CI build for us running Node v8.16.)

Comment (kellyselden) [reply package.json]: oops, that should have been "^8.3". Can you submit a fix?

**Comment by amyrlam** [comment]
Possible to bump the parent package https://github.com/kellyselden/ember-awesome-macros/blob/master/package.json#L32? I'm having the same issue in IE11. Downgraded to `ember-awesome-macros` 4.0.0 to fix in prod.

**Comment by kellyselden** [comment]
@amyrlam Yeah we can bump the minimum if you don't mind submitting.

**Comment by amyrlam** [comment]
@kellyselden thanks a lot! I'm thinking this is what you mean, let me know if needs edits? https://github.com/kellyselden/ember-awesome-macros/pull/565

**Pull request merged**